apidAnalytics_suite_test.go

```diff
@@ -220,7 +217,7 @@ func insertTestData(db apid.DB) {
 
 	txn, err := db.Begin()
 	Expect(err).ShouldNot(HaveOccurred())
-	txn.Exec("INSERT INTO APP_CREDENTIAL_APIPRODUCT_MAPPER (tenant_id,"+
+	txn.Exec("INSERT INTO kms_APP_CREDENTIAL_APIPRODUCT_MAPPER (tenant_id,"+
```

Comment:
I don't know the style guide, but what we did in other code is like "INSERT INTO kms_lowercase". It would be better if you can change it into this way.